component/authentication-endpoint/src/main/webapp/localstyles.css

```diff
@@ -0,0 +1,282 @@
+    background-image: -webkit-linear-gradient(top, #316a90, #4583ac);
+    background-image: -o-linear-gradient(top, #316a90, #4583ac);
+    background-image: -moz-linear-gradient(top, #316a90, #4583ac);
+    background-image: linear-gradient(top, #316a90, #4583ac);
+    background-repeat: repeat-x;
+    border-color: #0055cc #0055cc #003580;
+    border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25);
+    filter: progid:dximagetransform.microsoft.gradient(startColorstr='#316a90', endColorstr='#4583ac', GradientType=0);
+    filter: progid:dximagetransform.microsoft.gradient(enabled=false);
+}
+
+.login-header {
+    background: #e86d1f;
+    color: #fff;
+    padding: 5px;
+
+    -webkit-border-top-left-radius: 4px;
+    border-top-left-radius: 4px;
+    -moz-border-radius-topleft: 4px;
+
+    -webkit-border-top-right-radius: 4px;
+    border-top-right-radius: 4px;
+    -moz-border-radius-topright: 4px;
+
+    margin-top: 10px;
+}
+
+.form-horizontal .control-label {
+    width: 80px;
+}
+
+.form-horizontal .controls {
+    margin-left: 100px;
+}
+
+.form-horizontal .form-actions {
+    padding-left: 100px;
+    padding-bottom: 0;
+    margin-bottom: 0;
+}
+
+h1 {
+    margin-top: 20px;
+    color: #fff;
+    font-weight: normal;
+    font-size: 30px;
+}
+
+.different-login-container {
+    background: #F5F5F5;
+    padding: 10px;
+    -webkit-box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+    -moz-box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+    box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+}
+
+.main-login-container {
+    background: transparent url(../images/container-back.png) repeat left top;
+    padding: 20px 10px;
+    margin-bottom: 20px;
+    -webkit-box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+    -moz-box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+    box-shadow: 7px 7px 5px 0px rgba(50, 50, 50, 0.75);
+}
+
+.vertical-slitter {
+    border-right: solid 1px #fff;
+}
+
+.form-actions {
+    border-top: none;
+    background: transparent;
+}
+
+.btn-primary.active {
+    color: rgba(255, 255, 255, 0.75);
+}
+
+.btn-primary {
+    color: #ffffff;
+    text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
+    background: #45484d; /* Old browsers */
+    background: -moz-linear-gradient(top, #45484d 0%, #000000 100%); /* FF3.6+ */
+    background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #45484d), color-stop(100%, #000000)); /* Chrome,Safari4+ */
+    background: -webkit-linear-gradient(top, #45484d 0%, #000000 100%); /* Chrome10+,Safari5.1+ */
+    background: -o-linear-gradient(top, #45484d 0%, #000000 100%); /* Opera 11.10+ */
+    background: -ms-linear-gradient(top, #45484d 0%, #000000 100%); /* IE10+ */
+    background: linear-gradient(to bottom, #45484d 0%, #000000 100%); /* W3C */
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#45484d', endColorstr='#000000', GradientType=0); /* IE6-9 */
+    padding: 5px 10px;
+
```

Comment:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-token2/pull/2